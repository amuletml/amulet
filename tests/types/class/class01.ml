class show 'a begin
  val show : 'a -> string
end
