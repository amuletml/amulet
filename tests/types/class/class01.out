show : Req{'a : type}. constraint
show : Spec{'a : type}. show 'a => 'a -> string
